Blog: do more weird (#19230)
PostHog/posthog.com: contents/blog/on-doing-more-weird.mdx

---
date: 2026-08-04

> title: "Why and how we spend $45k/month on 'do more weird'"

Reviewer: ```suggestion
title: "Why and how we spend $45k/month on "do more weird""
```

Reviewer 2: I think quote marks will break it, but I think "doing more weird" is more parseable than "do more weird"

```suggestion
title: "Why and how we spend $45k/month on doing more weird"
```

I'm scared the $45k/month is sort of hiding how big the number is

```suggestion
title: "We spend $45,000 on doing more weird every month"
```

This one I don't think is better but it's an idea

```suggestion
title: "We spend $45,000 per month on doing more weird. Here's how and why"
```

author:
  - charles-cook
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
featuredImage: https://res.cloudinary.com/dmukukwp6/image/upload/mr_potato_hog_83221a5307.png
featuredImageType: full
category: Blog
tags:
  - Culture
  - Marketing
  - Founders
---

> 'Do more weird' is an actual PostHog [company value](/handbook/values#do-more-weird), and we have a discretionary $45k/mo budget that anyone can use to bring weird ideas to life. 'Operationalizing weirdness' sounds like an oxymoron that some soul-sucking MBA came up with (hey), but there are a ton of ways you can practically inject more weirdness into your company. 

Reviewer: ```suggestion
"Do more weird" is an actual PostHog [company value](/handbook/values#do-more-weird), and we have a discretionary $45k/mo budget that anyone can use to bring weird ideas to life. "Operationalizing weirdness" sounds like an oxymoron that some soul-sucking MBA came up with (hey), but there are a ton of ways you can practically inject more weirdness into your company. 
```

Let's get weird. 

## Why we do more weird in the first place

There are two answers to this. 

> The first is tactical - if you work at a startup, you are probably competing with much larger, better-known companies who can easily outspend you. Even if you are an extremely well-funded company, when you're going up against a publicly traded giant, your resources are probably minuscule in comparison. Sure you need to do all the regular growth playbook stuff, but to actually have an edge and stand out, you need to get weird. Take the example of a good old-fashioned marketing blog. Your competition here isn't your competitor's (presumably lame) blog, your competition is everything else that occupies your audience's attention on the internet - TikTok, online shopping, every good independent writer on Substack etc. Weirdness helps you cut through the noise. 

Reviewer 2: ```suggestion
The first is tactical - if you work at a startup, you are probably competing with much larger, better-known companies who can easily outspend you. Even if you are an extremely well-funded company, your resources are probably minuscule in comparison to a publicly traded giant. Sure, you need to do all the regular growth playbook stuff, but to actually stand out, you need to get weird. 

Take the example of a good old-fashioned marketing blog. Your competition here isn't your competitor's (presumably lame) blog, your competition is everything else that occupies your audience's attention on the internet - TikTok, online shopping, every writer on Substack, etc. Weirdness helps you cut through the noise.
```

> The second answer is more philosophical - the average person is just really weird! Don't believe me? Imagine if you just said every passing thought that was in your head out loud in an office - you probably wouldn't stick around for very long. Given people have so much weirdness swirling around in their brains anyway, why not harness that energy and make the places we work a little bit more fun? Sorry, I mean 'why not harness that energy and cynically exploit it in the service of shareholder value'. 

Reviewer: "people are already weird" doesn't feel like a strong reason alone. are we maybe saying something like "it's **easy to do** since people are already weird, so why not do it anyway"?

> The second answer is more philosophical - the average person is just really weird! Don't believe me? Imagine if you just said every passing thought that was in your head out loud in an office - you probably wouldn't stick around for very long. Given people have so much weirdness swirling around in their brains anyway, why not harness that energy and make the places we work a little bit more fun? Sorry, I mean 'why not harness that energy and cynically exploit it in the service of shareholder value'. 

Reviewer 2: ```suggestion
The second answer is more philosophical - the average person is just really weird! Don't believe me? Imagine if you just said every passing thought that was in your head out loud in an office - you probably wouldn't stick around for very long. Given people have so much weirdness swirling around in their brains anyway, why not harness that energy and make the places we work a little bit more fun? Sorry, I mean "why not harness that energy and cynically exploit it in the service of shareholder value."
```

<ImageSlider>

![Bus ad](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/busad_eca71ec131.jpeg)

![Website redesign](https://res.cloudinary.com/dmukukwp6/image/upload/websie_dd7fe0db65.png)

![Hedgehog Mode](https://res.cloudinary.com/dmukukwp6/image/upload/hedgehogmode_c78d753360.jpeg)

![DeskHog](https://res.cloudinary.com/dmukukwp6/image/upload/deskhog_1b816ae669.jpeg)

![Candle](https://res.cloudinary.com/dmukukwp6/image/upload/candle_76d58f66cc.jpeg)

</ImageSlider>

## Remember weirdness is relative

> Sometimes the thing you want to do just isn't _that_ weird, but that may still be ok depending on the context! For example, transparent pricing is weird in the context of how we bill customers, but it wouldn't make sense to do something truly weird like bartering grain for PostHog credit or something.

Reviewer 2: "If anyone wants to barter grain for PostHog credits, let me know." with a screenshot of this text is going to hit as a social post 

```suggestion
Sometimes the thing you want to do just isn't _that_ weird, but that may still be ok depending on the context. For example, transparent pricing is weird in the context of how we bill customers, but it wouldn't make sense to do something truly weird like bartering grain for PostHog credit or something.
```

On the other hand, the bar for weirdness in a marketing campaign is extremely high, because the world is full of marketing teams trying to do the same thing. A wry smile in response to the idea will not cut it. 

> Perhaps something in the middle would be a small easter egg in the settings page of your app. Weirdness exists on a spectrum. 

Reviewer 2: I think we need one more middle spectrum example here. 

```suggestion
Something in the middle would be a small easter egg in your settings page or making a [fairy tale version of a data processor agreement](/dpa). Weirdness exists on a spectrum. 
```

## How to do weird wrong

Before we do weird right, first we must avoid doing it wrong:

1. Don't make weirdness your marketing team's responsibility. They may be the _primary_ facilitators, but if weirdness is delegated to your marketing team, you'll eventually gravitate to a bunch of corporate try hard. 

> 2. Make the weirdness coherent with your brand. If you have a brand that feels very premium and high quality, weirdness may be an unusual spin on your sales process that delights customers, or a unique take on an event. 

Reviewer: ```suggestion
2. Make the weirdness coherent with your brand. If you have a brand that feels very premium and high quality, weirdness may take shape as an unusual spin on your sales process that delights customers, or a unique take on an event. 
```

> 3. In jokes are funny internally. They are not funny to your users. If you're sure, ask a friend externally who matches your ICP - 'I hate it' is ok, but 'I don't get it' is not. 

Reviewer: ```suggestion
3. Inside jokes are funny internally. They are not funny to your users. If you're not sure, ask a friend externally who matches your ICP – "I hate it" is okay, but "I don't get it" is not. 
```

> 4. Don't forget that it's a numbers game. If you pitch a few ideas or ship something that doesn't land, don't give up! What you see at PostHog is just a tiny fraction of the many terrible ideas we've had internally. 

Reviewer 2: One thing maybe add, do more weird is not just "lets have a cool merch line with funny items." Merch is a lot of work, we've cut down on the custom one off items.

## How to operationalize weird

> Assuming you are bought in and excited to do weird stuff, there are a few specific things we do at PostHog to help weird ideas become a real, shipped thing. 

Reviewer 2: One thing I can foresee being push back on this one is people being like "I want to do more weird, but I'm just a small fish"

- It's easier to do more weird early. I think it would be a lot harder to introduce do more weird in PostHog's culture now.
- There is cultural safety at PostHog, I don't feel that if I say something too weird or even inappropriate I will be fired for example, I know I'll get feedback and be able to calibrate myself for next time. 
- We're really bought in, it's a cultural value and execs (Charles) support it. 
- We're willing to have some duds. A lot of billboards we've run have had zero people comment on them. We've had easter eggs that like 3 people have found. We've wasted money on merch people have never worn. Our [documentary](https://www.youtube.com/watch?v=dGaSxBs3OsU) we spent a ton of time and money on has 2k views.

> We have a dedicated `#do-more-weird` Slack channel where anyone can propose any idea, no matter how ludicrous. If nobody engages with your idea, chances are it wasn't a good one - and that's ok!

Reviewer: ```suggestion
We have a dedicated `#do-more-weird` Slack channel where anyone can propose any idea, no matter how ludicrous. If nobody engages with your idea, chances are it wasn't a good one - and that's okay!
```

> Once a month, I have an automated skill scrape the last 30 days of ideas and pitch which ones should get support and why. It takes into account how much they are likely to cost, the effort, and how enthusiastic the response from the team was. It uses [this handbook page](/handbook/company/do-more-weird) as context for what makes a good candidate. A small (2-3 person) group then reviews the pitches and decides how to spend the money. 

Reviewer: ```suggestion
Once a month, I have an automated skill scrape the last 30 days of ideas and pitch which ones should get support and why. It takes into account how much they are likely to cost, how much effort they would take, and how enthusiastic the response from the team was. It uses [this handbook page](/handbook/company/do-more-weird) as context for what makes a good candidate. A small (2-3 person) group, aka The Council of Weird, then reviews the pitches and decides how to spend the money. 
```

Reviewer 2: We should also say "council of weird" because that is a good name

> Of course, many weird things don't actually require money - in those cases, we just encourage the person who pitched it to do the thing! There's nothing like basking in the warm glow of your team mates for shipping something like [DeskHog](/deskhog) or [Hedgehog Mode](/sparks-joy/hedgehog-mode). 

Reviewer: ```suggestion
Of course, many weird things don't actually require money - in those cases, we just encourage the person who pitched it to do the thing. There's nothing like basking in the warm glow of your teammates for shipping something like [DeskHog](/deskhog) or [Hedgehog Mode](/sparks-joy/hedgehog-mode). 
```

Reviewer 2: Maybe a hackathon mention with mention of: https://posthog.com/newsletter/hackathons

---

> date: 2026-08-04

Author: ```suggestion
date: 2026-08-05
```

  - Founders
---

"Do more weird" is an actual PostHog [company value](/handbook/values#do-more-weird), and we have a discretionary $45k/mo budget that anyone can use to bring weird ideas to life. "Operationalizing weirdness" sounds like an oxymoron that some soul-sucking MBA came up with (hey), but there are a ton of ways you can practically inject more weirdness into your company. 

Let's get weird. 

## Why we do more weird in the first place

There are two answers to this. 

The first is tactical - if you work at a startup, you are probably competing with much larger, better-known companies who can easily outspend you. Even if you are an extremely well-funded company, your resources are probably minuscule in comparison to a publicly traded giant. Sure, you need to do all the regular growth playbook stuff, but to actually stand out, you need to get weird. 

Take the example of a good old-fashioned marketing blog. Your competition here isn't your competitor's (presumably lame) blog, your competition is everything else that occupies your audience's attention on the internet - TikTok, online shopping, every writer on Substack, etc. Weirdness helps you cut through the noise.

> The second answer is more philosophical - the average person is just really weird! Don't believe me? Imagine if you just said every passing thought that was in your head out loud in an office - you probably wouldn't stick around for very long. Given people have so much weirdness swirling around in their brains anyway, why not harness that energy and make the places we work a little bit more fun? Sorry, I mean "why not harness that energy and cynically exploit it in the service of shareholder value."

Author: ```suggestion
The second answer is more philosophical - the average person is already really weird, so why not do this stuff anyway! Don't believe me? Imagine if you just said every passing thought that was in your head out loud in an office - you probably wouldn't stick around for very long. Given people have so much weirdness swirling around in their brains anyway, why not harness that energy and make the places we work a little bit more fun? Sorry, I mean "why not harness that energy and cynically exploit it in the service of shareholder value."
```

It is extremely helpful to have a founder or executive sponsor for weird things. Someone who openly encourages it, but is also willing to have some duds (oh boy, we've had some duds). A lot of the billboards we've run have had zero people comment on them. We've had easter eggs that ~3 people have found. We've wasted money on merch people have never worn. Our [documentary](https://www.youtube.com/watch?v=dGaSxBs3OsU) that we spent a ton of time and money on has 2k views.

Individual in a small company? It's probably easy to convince people. 
Individual in a larger company? Find your executive sponsor. 
Leader in a larger company? Be the sponsor!

## How to operationalize weird

Assuming you are bought in and excited to do weird stuff, there are a few specific things we do at PostHog to help weird ideas become a real, shipped thing. 

We have a dedicated `#do-more-weird` Slack channel where anyone can propose any idea, no matter how ludicrous. If nobody engages with your idea, chances are it wasn't a good one - and that's okay!

> Once a month, I have an automated skill scrape the last 30 days of ideas and pitch which ones should get support and why. It takes into account how much they are likely to cost, how much effort they would take, and how enthusiastic the response from the team was. It uses [this handbook page](/handbook/company/do-more-weird) as context for what makes a good candidate. A small (2-3 person) group, aka The Council of Weird, then reviews the pitches and decides how to spend the money. 

Author: ```suggestion
Once a month, I have an automated skill scrape the last 30 days of ideas and pitch which ones should get support and why. It takes into account how much they are likely to cost, how much effort they would take, and how enthusiastic the response from the team was. It uses [this handbook page](/handbook/company/do-more-weird) as context for what makes a good candidate. A small (2-3 person) group, aka The Council of Weird, then reviews the pitches and decides how to spend the money. When it comes to setting the budget, try 5% of your overall marketing budget first and go from there. 
```
